src/test/resources/simple/not-allowed-url-bag/fetch.txt

```diff
@@ -0,0 +1 @@
+ftp://dans.knaw.nl/nl/over/organisatie-beleid/informatiemateriaal/Software_Sustainability_DANS_NLeSC_2016.pdf  555292  data/brochure/Software_Sustainability_DANS_NLeSC_2016.pdf
```

Comment:
Can we use another file here that we ourselves have more control over? Our tests will unexpectedly fail as soon as this file gets removed or replaced or modified!

@janvanmansum maybe put this file on teasy/aeasy?

@vesaakerman have you tested this with no internet? Maybe ask @jo-pol how to do this.